q2_types/per_sample_sequences/tests/test_format.py

```diff
@@ -200,5 +202,58 @@ def test_slanepsample_paired_end_fastq_dir_fmt_validate_negative(self):
             format.validate()
 
 
+class TestQIIME1DemuxFormat(TestPluginBase):
+    package = 'q2_types.per_sample_sequences.tests'
+
+    def setUp(self):
+        super().setUp()
+
+        self.positives = [
+            'short.fna', 'long.fna', 'single-record.fna',
+            'with-descriptions.fna'
```

Comment:
Could we add a sample file from `split_libraries.py` with the barcode stuff in the description? I'm sure it'll work, but since this is also an import path for users of QIIME 1, it seems like we should have a test of it.

Reply:
Great idea!